app/controllers/application_controller.rb

```diff
@@ -90,4 +90,14 @@ def set_visitor_props
       @visitor_properties = {user: current_user.email, role: current_user.user_role}
     end
   end
+
+  def log_spam_attempt(form)
+    ip_address = request.remote_ip
+    url_orig = request.original_url
+    email = params["email"]
+    subject = params["subject"]
+    message = params["message"]
+    phone = params["phone"]
+    FormSpam.create(form: form, original_url: url_orig, ip_address: ip_address, email: email, subject: subject, message: message, phone: phone)
```

Comment:
@JoshingYou1 @bradjohnson92008 I'm wondering how useful it would be for us to store potentially malicious user input in our DB? Is it helpful for us to know what the spammer was trying to input or that there was an attempt to input spam? 🤔

If the latter, then I would only save the IP address and request url.

What are your two's thoughts?

Reply:
Thats a good point.. I thought everyone would want more info than less..  but your right... it may just be junk filling up our db...  I could also add a counter column and increment that... if the same IP address hits multiple times.. instead of creating a new record.. just increment the counter.  Whatever you guys think..  I'm just interested to see if it catches anything..  feel like i'm going fishing.. lol.

Reply:
Cool so I removed all the form fields... no reason to store the junk.  most likely if records show up here we'll just need the ip_address and url.